cli/app.go

```diff
@@ -242,7 +248,7 @@ var app = &cli.App{
 				{
 					Name:      "export",
 					Usage:     "download data from Viam cloud",
-					UsageText: createUsageText("viam data export", []string{dataFlagDestination, dataFlagDataType}, true),
+					UsageText: createUsageText("data export", []string{dataFlagDestination, dataFlagDataType}, true),
```

Comment:
I now always append `viam` to the front of the `UsageText` in `createUsageText`, so no need to include it here.